meteor-app/imports/startup/client/routes.js

```diff
@@ -95,8 +102,8 @@ FlowRouter.route('/workspace', {
         <WorkspacePage
           {...{
             store,
-            updateFilterValue: (newValue) => {
-              FlowRouter.go(path, {}, {
+            updateURL: (newValue) => {
```

Comment:
This function signature does not match what it does. If you think about it, what would people expect `updateUrl(value)` to do? Let's assume it "updates the url" but how? And with just one value?